Library/ViewModels/LiveStreamContainerViewModel.swift

```diff
@@ -214,14 +214,12 @@ LiveStreamContainerViewModelInputs, LiveStreamContainerViewModelOutputs {
 
     let hideWhenReplay = Signal.merge(
       project.mapConst(true),
-      event.map { !$0.liveNow },
-      self.showErrorAlert.mapConst(true)
```

Comment:
To be honest we probably don't even need the `project.mapConst(true)` anymore either as we know immediately if it's live or a replay. I'll look at maybe removing that for simplification.